x-pack/plugins/elastic_assistant/server/lib/langchain/helpers.ts

```diff
@@ -31,27 +31,3 @@ export const getMessageContentAndRole = (prompt: string): Pick<Message, 'content
   content: prompt,
   role: 'user',
 });
-
-export interface ResponseBody {
-  status: string;
-  data: Record<string, unknown>;
-  connector_id: string;
-}
-
-/** An unsafe, temporary stub that parses assistant messages from the request with no validation */
-export const unsafeGetAssistantMessagesFromRequest = (
```

Comment:
Thank you for this cleanup and adding validation @stephmilovic! <img width="16" src="https://user-images.githubusercontent.com/2946766/141219243-86619f7f-c300-4355-96f5-316d70aa6a0e.png" />